src/navigation/mobileStepper/mobileStepper.jsx

```diff
@@ -0,0 +1,119 @@
+import ClassNames from 'classnames';
+import MUIMobileStepper from '@material-ui/core/MobileStepper';
+import PropTypes from 'prop-types';
+import React from 'react';
+import {
+    BEM_MOBILE_STEPPER,
+    UI_CLASS_NAME,
+} from '../../global/constants';
+import makeStyles from '../../styles/makeStyles';
+
+const propTypes = {
+    /**
+     * Set the active step (zero based index).
+     * Defines which dot is highlighted when the variant is 'dots'.
+     */
+    activeStep: PropTypes.number,
+    /**
+     * A back button element. For instance, it can be a `Button` or an `IconButton`.
+     */
+    backButton: PropTypes.node,
+    /**
+     * Override or extend the styles applied to the component.
+     * See [CSS API](#css) below for more details.
+     */
+    classes: PropTypes.shape({}),
+    /**
+     * @ignore
+     */
```

Comment:
What's the point of `@ignore` directive on these comments?
Is this trying to tell you not to use `className` prop?